File: Examples/DotNET/SDK/CSharp/Watermark/RemoveWatermark.cs
using System;
using Aspose.Cloud;

namespace CSharp.Watermark
{
    class RemoveWatermark
    {
        public static void Run()
        {
            string inputfile = "doc-sample-watermark.doc"; 

            string outfile = Common.GetOutputFilePath(inputfile, true);
            
            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Remove Watermark from a Word Document
            Common.WordsService.WordsDocumentWatermark.DeleteWatermark(inputfile, outfile, Common.FOLDER);

            // Download output file from cloud storage and save on local directory
            string dataDir = Common.DownloadFile(inputfile, string.Empty);

            Console.WriteLine("\nWatermark text added successfully.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/DotNET/SDK/VisualBasic/Watermark/RemoveWatermark.vb
Imports Aspose.Cloud

Namespace Watermark
    Class RemoveWatermark
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample-watermark.doc"

            Dim outfile As String = Common.GetOutputFilePath(inputfile, True)

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Remove Watermark from a Word Document
            Common.WordsService.WordsDocumentWatermark.DeleteWatermark(inputfile, outfile, Common.FOLDER)

            ' Download output file from cloud storage and save on local directory
            Dim dataDir As String = Common.DownloadFile(inputfile, String.Empty)

            Console.WriteLine(Convert.ToString(vbLf & "Watermark text added successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace